### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by rahul-kr01**

### CODE REVIEW

File: src/app/core/services/local-storage.service.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+@Injectable({
+  providedIn: 'root'
+})
+export class LocalStorageService {
+
+  constructor() { }
+
+  public saveLocalData(key: string, value: string): Promise<any> {
+    return new Promise((resolve,reject) => {
+      let data = localStorage.setItem(key, value)
```
Comment (kiranharidas187) [inline src/app/core/services/local-storage.service.ts:14]: did you test if these functions are working??

File: src/app/core/services/local-storage.service.spec.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
```
Comment (kiranharidas187) [inline src/app/core/services/local-storage.service.spec.ts:1]: inside services, create a folder named local-storage and put these files inside that

File: src/app/core/services/local-storage.service.ts (hunk 1 above)
Comment (rahul-kr01) [reply src/app/core/services/local-storage.service.ts:14]: yes

File: src/app/core/services/local-storage.service.spec.ts (hunk 2 above)
Comment (rahul-kr01) [reply src/app/core/services/local-storage.service.spec.ts:1]: sure

**Pull request merged**